src/Astreiko.Homework8/Program.cs

```diff
@@ -29,7 +29,7 @@ static void Main(string[] args)
 
                     new Thread(shop.CheckQueueCustomers).Start();
 
-                    shop.StartVisitorsGenerator();
+                    new Thread(shop.StartVisitorsGenerator).Start();
```

Comment:
This will be a foreground thread, do you really need to wait for this generation to complete?